src/index.js

```diff
@@ -81,6 +83,7 @@ import './style.scss';
  * @property {string[]} [disallowEmbed] - List of embed names to remove
  * @property {object[]} [customStores] - Array of custom stores
  * @property {boolean} [footer] - Show footer component
+ * @property {history} [history] - Enable hisotry undo/redo functionality
```

Comment:
```suggestion
 * @property {boolean} [history] - Enable history undo/redo functionality
```

Are we going to use this prop in a future PR?

Reply:
Sorry, this is a leftover from another approach I tried before the final one. I'm removing this. Thanks!